lib/vector-buffers/src/variants/disk_v2/tests/invariants.rs

```diff
@@ -610,13 +614,18 @@ async fn writer_waits_for_reader_after_validate_last_write_fails_and_data_file_s
         let data_dir = dir.to_path_buf();
 
         async move {
-            let record_size: u32 = 32;
-
-            // Create our buffer with an arbitrarily low max data file size, which will let us
-            // quickly run through the file ID range.  We want to be able to write at least two
-            // records to each data file, though.
+            let record_size = 42;
+            let record = SizedRecord(record_size);
+            let corrected_record_size = get_corrected_max_record_size(&record);
+            let max_data_file_size = (corrected_record_size * 2)
+                .try_into()
+                .expect("Vector does not supports 128-bit architectures.");
```

Comment:
This exception is a bit misleading, as it requires both an 128-bit architecture and a value of at least 2^64, which is rather impossible for a record size of 42.

Reply:
Fair. I'll make it more relevant to the context/callsite.

Reply:
Addressed in c54ae438f.